docs/ContactUs.adoc

```diff
@@ -1,6 +1,6 @@
 = Contact Us
 :stylesDir: stylesheets
 
-* *Bug reports, Suggestions* : Post in our https://github.com/se-edu/addressbook-level4/issues[issue tracker] if you noticed bugs or have suggestions on how to improve.
+* *Bug reports, Suggestions* : Post in our https://github.com/CS2103AUG2017-W15-B1/Main/issues[issue tracker] if you noticed bugs or have suggestions on how to improve.
 * *Contributing* : We welcome pull requests. Follow the process described https://github.com/oss-generic/process[here]
-* *Email us* : You can also reach us at `damith [at] comp.nus.edu.sg`
+* *Email us* : You can also reach us at `E0031757@u.nus.edu.sg (Jackie)`
```

Comment:
use `E0031757 [at] u.nus.edu` instead, so as to prevent spam bots from picking up your email address

Reply:
and no `.sg` for students

Reply:
Thanks, i have made the changes